Dockerfile

```diff
@@ -42,29 +42,23 @@ RUN apt-get update \
     && rm -rf /var/lib/apt/lists/*
 
 
-ENV HUSARION_ROS_BUILD_TYPE=simulation
-
 WORKDIR /ros2_ws
 RUN git clone https://github.com/frankaemika/franka_ros2.git src/franka_ros2
-RUN git clone https://github.com/husarion/husarion_ugv_ros.git src/husarion_ugv_ros
+RUN git clone https://github.com/husarion/husarion_ugv_ros.git husarion_ugv_ros
+RUN mv husarion_ugv_ros/husarion_ugv_description src/husarion_ugv_description
+RUN mv husarion_ugv_ros/husarion_ugv_msgs src/husarion_ugv_msgs
+RUN mv husarion_ugv_ros/husarion_ugv src/husarion_ugv
+RUN rm -rf husarion_ugv_ros
 COPY ./husarion_ugv_franka_manipulator_bringup src/husarion_ugv_franka_manipulator_bringup
 
-# RUN vcs import src < src/franka.repos --recursive --skip-existing \
-#     && sudo apt-get update \
-#     && rosdep update \
-#     && rosdep install --from-paths src --ignore-src --rosdistro $ROS_DISTRO -y \
-#     && sudo apt-get clean \
-#     && sudo rm -rf /var/lib/apt/lists/* \
-#     && rm -rf /home/$USERNAME/.ros \
-#     && rm -rf src \
-#     && mkdir -p src
 
 RUN apt-get update  && \
     apt-get install -y \
         ros-dev-tools && \
     # Setup workspace
     vcs import src < src/franka_ros2/franka.repos --recursive --skip-existing && \
-    vcs import src < src/husarion_ugv_ros/husarion_ugv/${HUSARION_ROS_BUILD_TYPE}_deps.repos && \
+    vcs import src < src/husarion_ugv/${HUSARION_ROS_BUILD_TYPE}_deps.repos && \
```

Comment:
This will also clone behaviortree.cpp and husarion_controllers. Maybe just git clone ros_components_description?